docs/core/deploying/trim-warnings/il2045.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "IL2045: Custom attribute is referenced in code but the trimmer was instructed to remove all of its instances"
+description: "Learn about trim warning IL2045: CustomAttributeAnnotationsRemovingUsedAttribute"
+ms.date: 08/25/2021
+ms.topic: reference
+author: mateoatr
+ms.author: matorre
+f1_keywords:
+  - "IL2045"
+---
+# IL2045: Custom attribute is referenced in code but the trimmer was instructed to remove all of its instances"
+
+## Cause
+
+A custom attribute that is being referenced in code was instructed for removal by the trimmer.
```

Comment:
I think we should add an explanation that in this case the trimmer will typically remove the attribute usage (instantiation), but keep the attribute type itself. Meaning the trimmer will very likely break the code using that attribute type.

From the current sentence it's not clear what the trimmer does with the inconsistency.